admin/translations/fr.json

```
@@ -319,7 +319,8 @@
       "membership-item": {
         "actions": {
           "edit-role": "Modifier le rôle"
-        }
+        },
+        "last-connection-default-date": "Non connecté depuis 03/25"
```

2 remarques : 
* Concernant la date, le format à utiliser est `MM/YYYY`, ce qui donne : `03/2025`
* Concernant le nom de la clé de traduction, il ne s'agit pas d’une « defaultDate », mais plutôt d'un message d’information, on gagnerait donc à utiliser une clé de traduction comme : `no-last-connection-date-info`

```suggestion
        "no-last-connection-date-info": "Non connecté depuis 03/2025"
```